### PR TITLE
tracelLogExclude to tracerLogExclude

### PR DESCRIPTION
**Pull request by wenpos**
fix a variable name for misspelling "tracerLogExclude" to "tracelLogExclude"

### CODE REVIEW

**Comment by bleskes** [comment]
As the person who made the mistake - thank you for fixing :) I'll merge this in.

File: core/src/main/java/org/elasticsearch/transport/TransportService.java [hunk 1]
```diff
@@ -174,7 +174,7 @@ void setTracerLogInclude(List<String> tracerLogInclude) {
     }
 
     void setTracerLogExclude(List<String> tracelLogExclude) {
```
Comment (jasontedor) [inline core/src/main/java/org/elasticsearch/transport/TransportService.java]: This should be changed to `tracerLogExclude` as well.

**Review by jasontedor — COMMENTED** [review]
I left a comment regarding another typo.

**Comment by jasontedor** [comment]
Oh, this is already merged. I'll just push directly.

**Comment by jasontedor** [comment]
I pushed 9bb8bd05780d657ead3ce9335f6b66903b759bb5 to master and 942d6d558962092481fe604f9b072d6630859f41 to 5.x.

**Comment by wenpos** [comment]
@bleskes @jasontedor thanks

**Pull request merged**